libs/designer-ui/src/lib/html/plugins/toolbar/ColorPickerTextInput.tsx

```diff
@@ -26,7 +27,12 @@ export function TextInput({ label, value, onChange, placeholder = '', 'data-test
 
   return (
     <div className="msla-colorpicker-input-wrapper">
-      <label className="msla-colorpicker-input-label">{label}</label>
+      <label
```

Comment:
Can we convert this to a V9 component so it just colors passively?

Reply:
Think this dropdown also gets rendered outside, so when using a label we won't be able to get the provider. However, Eric from PA is working/ has a PR for moving these dropdowns to fluent ui, so this won't be the case for very long